apps/voting/contracts/Voting.sol

```diff
@@ -453,16 +676,64 @@ contract Voting is IForwarder, AragonApp {
     }
 
     /**
-    * @dev Internal function to check if a voter can participate on a vote. It assumes the queried vote exists.
+    * @dev Internal function to check if the vote is open and given option is applicable at the current phase.
+    *      It assumes the queried vote exists.
+    * @param vote_ The queried vote
+    * @param _supports Whether the voter supports the vote or not
     * @return True if the given voter can participate a certain vote, false otherwise
     */
-    function _canVote(uint256 _voteId, address _voter) internal view returns (bool) {
-        Vote storage vote_ = votes[_voteId];
-        return _isVoteOpen(vote_) && token.balanceOfAt(_voter, vote_.snapshotBlock) > 0;
+    function _isValidPhaseToVote(Vote storage vote_, bool _supports) internal view returns (bool) {
```

Comment:
We never do postfix `_` in this repo, it's not a blocker but I really don't want to mix different code styles

Reply:
My bad, postfix is ok